lib/libc/picolibc/CMakeLists.txt

```diff
@@ -22,4 +22,19 @@ if(NOT CONFIG_PICOLIBC_USE_MODULE)
     zephyr_link_libraries(-DPICOLIBC_INTEGER_PRINTF_SCANF)
   endif()
 
+  execute_process(
+    COMMAND echo "#include <picolibc.h>"
+    COMMAND ${CMAKE_C_COMPILER} --specs=picolibc.specs -M -E -
```

Comment:
same problem here wrt. Windows.

Reply:
let's hope the same temp file plan works here too.